server/controllers/work.js

```diff
@@ -29,14 +29,15 @@ export const work = async(ctx, next) => {
   const singleWork = await getWork(id);
   const miroId = singleWork.identifiers[0].value;
   const imgLink = imageUrlFromMiroId(miroId, shouldUseIiif(ctx));
+  const urlWithoutQuery = ctx.originalUrl.split('?')[0];
```

Comment:
I think we can get this here?
https://github.com/koajs/koa/blob/master/docs/api/request.md